src/app/api/stacks/services/account-service.ts

```diff
@@ -0,0 +1,16 @@
+import type { AddressBalanceResponse } from '@stacks/stacks-blockchain-api-types';
+
+import type { DefaultNetworkConfigurations } from '@shared/constants';
+
+// import type { AddressBalanceResponse } from '@shared/models/account.model';
```

Comment:
I removed this and am using the one imported from the api-types bc the need for this might be outdated? I'm not seeing why we would need to keep this?